packages/create-toolpad-app/tsconfig.json

```diff
@@ -10,7 +10,6 @@
     "esModuleInterop": true,
     "skipLibCheck": true,
     "noEmit": false,
-    "rootDir": "src",
```

Comment:
Why is this line to be removed?

Reply:
https://github.com/JerryWu1234/mui-toolpad/blob/653d82fc1ce5c7c82471237b6e54f3bafcdaadae/packages/create-toolpad-app/src/index.ts#L12
because of this.
I need to get the upper package.json,

I’m a little confused about what the full name of DX is.

Reply:
"DX" usually refers to developer experience, unless you're referring to it from some other specific context.

Each folder inside `packages` is a separate "project", so you must not use the `package.json` of one project inside another. Instead, we can create a constant `create-toolpad-app` which contains the minimum supported node version and compare against that.

Reply:
such as const TOOLPADAPPNODEVERSION = "minimum version"?

related to an issue https://github.com/mui/mui-toolpad/issues/2092#issuecomment-1567937877
 
I sorry I'm not clear expression, actually, I only use the package.json's `engines.node` in the create-toolpad-app.

Reply:
I think I'd also prefer to keep `rootDir` as is. It ensures a consistent dist folder layout. How would you feel about just reading the `package.json` file from the filesystem using something like [`readJsonFile`](https://github.com/mui/mui-toolpad/blob/bf9a2c17adc921ef614d606d24b1aeaea9d05a2f/packages/toolpad-app/src/utils/fs.ts#L11)?

Reply:
> I think I'd also prefer to keep `rootDir` as is. It ensures a consistent dist folder layout. How would you feel about just reading the `package.json` file from the filesystem using something like [`readJsonFile`](https://github.com/mui/mui-toolpad/blob/bf9a2c17adc921ef614d606d24b1aeaea9d05a2f/packages/toolpad-app/src/utils/fs.ts#L11)?
> 
> With "DX" I referred to [developer experience](https://microsoft.github.io/code-with-engineering-playbook/developer-experience/). I should probably just have said UX here as I was referring to the user installing `toolpad` (a user who happens to be a developer). I updated my comment.

thanks your suggestion ,I will do it lete